v2/docs/gs/params.md

```diff
@@ -150,15 +177,15 @@ The following KVS create-time parameters are part of the stable API.
 | :-- | :-- | :-- |
 | `prefix.length` | `0` | Key prefix length (bytes) |
 
-!!! tip
+!!! info
     The KVS name `default` is reserved and may not be used in
     `hse_kvdb_kvs_create()` API calls or with the CLI.
```

Comment:
I wonder if we have tests for this...

Reply:
Good question.  @janivishalm can you confirm?

Reply:
We don't have this test. I'll add this part of my upcoming sqa PR for pmem tests, and then to meson test suite